datahub-web-react/src/app/shared/RoutedTabs.tsx

```diff
@@ -3,6 +3,14 @@ import { Route, Switch, useRouteMatch, useLocation } from 'react-router-dom';
 import { Redirect, useHistory } from 'react-router';
 import { Tabs } from 'antd';
 import { TabsProps } from 'antd/lib/tabs';
+import styled from 'styled-components';
```

Comment:
Just make sure the other places where we use RoutedTabs are NOT affected.

This includes Dataset profile, User Profile, Group Profile, and Ingestion tab. 

If they all look the same as before, we can merge

Reply:
Worked on it and committed my changes